analytics/modules/agma/module.go

```diff
@@ -0,0 +1,54 @@
+package agma
+
+import (
+	"encoding/json"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/prebid/prebid-server/v3/analytics"
+	"github.com/prebid/prebid-server/v3/analytics/analyticsdeps"
+	"github.com/prebid/prebid-server/v3/config"
+)
+
+type Config struct {
+	Enabled  bool                             `mapstructure:"enabled" json:"enabled"`
+	Endpoint config.AgmaAnalyticsHttpEndpoint `mapstructure:"endpoint" json:"endpoint"`
+	Buffers  struct {
+		EventCount int    `mapstructure:"eventCount" json:"eventCount"`
+		BufferSize string `mapstructure:"bufferSize" json:"bufferSize"`
+		Timeout    string `mapstructure:"timeout" json:"timeout"`
+	} `mapstructure:"buffers" json:"buffers"`
+	Accounts []config.AgmaAnalyticsAccount `mapstructure:"accounts" json:"accounts"`
```

Comment:
Remove all of the `mapstructure` annotations. Make this same change for the other modules (pubstack and filelogger).